action.yaml

```diff
@@ -21,6 +21,10 @@ inputs:
     description: 'Pick output formatters.'
     required: false
     default: 'github_status github_pr'
+  path:
+    description: 'Relative path to check'
+    required: false
+    default: '.'
```

Comment:
default is `.`
https://github.com/prontolabs/pronto/blob/52c6a771cb5e6c3274b3f7d3f60122fb2f1c9d69/lib/pronto/cli.rb#L48